Subject: CLEVO M360S acpi irq workaround

Some Hardware vendor use CLEVO M360S Laptop OEM.
This acpi IRQ routing is not correct. we use normal IRQ routing for default.

Signed-off-by: Go Taniguchi <go@turbolinux.co.jp>

---
 arch/x86/kernel/acpi/boot.c |    8 ++++++++
 1 file changed, 8 insertions(+)

--- a/arch/x86/kernel/acpi/boot.c
+++ b/arch/x86/kernel/acpi/boot.c
@@ -1615,6 +1615,14 @@ static struct dmi_system_id __initdata a
 		     },
 	 },
 	{
+	 .callback = disable_acpi_irq,
+	 .ident = "CLEVO M360S",
+	 .matches = {
+		     DMI_MATCH(DMI_SYS_VENDOR, "CLEVO Co."),
+		     DMI_MATCH(DMI_PRODUCT_NAME, "M360S"),
+		     },
+	 },
+	{
 		/*
 		 * Latest BIOS for IBM 600E (1.16) has bad pcinum
 		 * for LPC bridge, which is needed for the PCI
